pkg/controllers/uiplugin/logging.go

```diff
@@ -0,0 +1,129 @@
+package uiplugin
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	osv1alpha1 "github.com/openshift/api/console/v1alpha1"
+	"gopkg.in/yaml.v3"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	uiv1alpha1 "github.com/rhobs/observability-operator/pkg/apis/uiplugin/v1alpha1"
+)
+
+func createLoggingPluginInfo(plugin *uiv1alpha1.UIPlugin, namespace, name, image string, features []string) (*UIPluginInfo, error) {
+	loggingConfig := plugin.Spec.Logging
+	if loggingConfig == nil {
+		return nil, fmt.Errorf("logging configuration can not be empty for plugin type %s", plugin.Spec.Type)
+	}
+
+	if loggingConfig.LokiStack.Name == "" {
+		return nil, fmt.Errorf("LokiStack name can not be empty for plugin type %s", plugin.Spec.Type)
+	}
+
+	configYaml, err := marshalLoggingPluginConfig(loggingConfig)
+	if err != nil {
+		return nil, fmt.Errorf("error creating plugin configuration file: %w", err)
+	}
+
+	extraArgs := []string{
+		"-plugin-config-path=/etc/plugin/config/config.yaml",
+	}
+
+	if len(features) > 0 {
+		extraArgs = append(extraArgs, fmt.Sprintf("-features=%s", strings.Join(features, ",")))
+	}
+
+	pluginInfo := &UIPluginInfo{
+		Image:             image,
+		Name:              name,
+		ConsoleName:       "logging-view-plugin",
+		DisplayName:       "Logging View",
+		ExtraArgs:         extraArgs,
+		ResourceNamespace: namespace,
+		Proxies: []osv1alpha1.ConsolePluginProxy{
+			{
+				Type:      "Service",
+				Alias:     "backend",
+				Authorize: true,
+				Service: osv1alpha1.ConsolePluginProxyServiceConfig{
+					Name:      fmt.Sprintf("%s-gateway-http", plugin.Spec.Logging.LokiStack.Name),
+					Namespace: "openshift-logging", // TODO decide if we want to support LokiStack in other namespaces
+					Port:      8080,
```

Comment:
Is the port fixed for a lokiStack?

Reply:
Yes, the Loki Operator currently provides no way to customize the port.